WORKSPACE.bazel

```diff
@@ -141,7 +141,7 @@ bazel_skylib_workspace()
 
 
 # Load iceoryx rules
-ICEORYX_VERSION = "2.95.5"
+ICEORYX_VERSION = "2.95.6"
 
 maybe(
     name = "iceoryx",
```

Comment:
Oh, I forgot about the sha256. Could you please change `b68d0603ca39a852db5d12d8e83b0c7b560acdbe8d4cbcdfca1a07ada433bdd5` to `745927c5e5e034cd840c1a18dff217818c042507b814d7bf001bcc61bcf5281b` to fix the bazel issue.